audit_vms.py

```diff
@@ -45,7 +45,7 @@ def main(argv):
             print('\n'.join('{} - uptime: {}d and {:.1f}h'.format(vm_id, uptime_in_days, uptime_in_hours) for vm_id, uptime_in_days, uptime_in_hours in ghost_vms_ids))
             if args.delete_ghost_vms:
                 print('You are about to delete all those VMs.')
-                if not ask_for_confirmation():
+                if not ask_for_confirmation(args.force_delete):
```

Comment:
Pour éviter de passer un paramètre à `ask_for_confirmation`, le `if` pourrait aussi devenir celui-ci:
```python
if not (args.force_delete or ask_for_confirmation()):
```
Comme tu veux ;)

Reply:
Rhooo, super, merci !
J'avais testé plusieurs solutions et combinaisons à base de not et or sans succès. C'est tellement plus simple ^^